hw-02-collections-and-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -1,14 +1,17 @@
 package homework;
 
-@SuppressWarnings({"java:S1186", "java:S1135", "java:S1172"}) // при выполнении ДЗ эту аннотацию надо удалить
+import java.util.LinkedList;
+import java.util.List;
+
 public class CustomerReverseOrder {
 
-    // todo: 2. надо реализовать методы этого класса
-    // надо подобрать подходящую структуру данных, тогда решение будет в "две строчки"
+    List<Customer> customers = new LinkedList<>();
```

Comment:
```suggestion
    private final Deque<Customer> customers = new LinkedList<>();
```

Правильнее будет `Deque` и `private final`.